### PR TITLE
feat(npm): Update Yarn install arguments

### PR DESCRIPTION
**Pull request by kidonng**
According to https://github.com/renovatebot/renovate/issues/5230#issuecomment-680140287, this should fix issues with updating lock files with Yarn 2.

httpTimeout: https://yarnpkg.com/configuration/yarnrc/#httpTimeout

### CODE REVIEW

File: lib/manager/npm/post-update/yarn.ts [hunk 1]
```diff
@@ -74,13 +74,14 @@ export async function generateLockFile(
...
+    let cmdOptions = '';
     if (global.trustLevel !== 'high' || config.ignoreScripts) {
       if (isYarn1) {
-        cmdOptions += ' --ignore-scripts';
+        cmdOptions +=
+          ' --ignore-scripts --ignore-engines --ignore-platform --network-timeout 100000';
       } else {
         extraEnv.YARN_ENABLE_SCRIPTS = '0';
+        extraEnv.YARN_HTTP_TIMEOUT = '100000';
       }
```
Comment (viceice) [inline lib/manager/npm/post-update/yarn.ts:85]: should look like:

```ts
    let cmdOptions = '';
    if (isYarn1) {
      cmdOptions += '--ignore-engines --ignore-platform --network-timeout 100000';
    } else {
      extraEnv.YARN_HTTP_TIMEOUT = '100000';
    }
    if (global.trustLevel !== 'high' || config.ignoreScripts) {
      if (isYarn1) {
        cmdOptions += ' --ignore-scripts';
      } else {
        extraEnv.YARN_ENABLE_SCRIPTS = '0';
      }
```

**Review by viceice — CHANGES_REQUESTED** [review]

File: lib/manager/npm/post-update/yarn.ts (hunk 1 above)
Comment (kidonng) [reply lib/manager/npm/post-update/yarn.ts:85]: Certainly! I didn't notice the if clause

File: lib/manager/npm/post-update/yarn.ts [hunk 2]
```diff
@@ -74,8 +74,13 @@ export async function generateLockFile(
       preCommands.push(optimizeCommand);
     }
     const commands = [];
-    let cmdOptions =
-      '--ignore-engines --ignore-platform --network-timeout 100000';
+    let cmdOptions = '';
+    if (isYarn1) {
+      cmdOptions +=
+        ' --ignore-engines --ignore-platform --network-timeout 100000';
```
Comment (rarkins) [inline lib/manager/npm/post-update/yarn.ts:80]:
```suggestion
        '--ignore-engines --ignore-platform --network-timeout 100000';
```

**Review by rarkins — CHANGES_REQUESTED** [review]

File: lib/manager/npm/post-update/__snapshots__/yarn.spec.ts.snap [hunk 3]
```diff
@@ -3,7 +3,7 @@
 exports[`manager/npm/post-update/yarn catches errors 1`] = `
 Array [
   Object {
-    "cmd": "yarn install --ignore-engines --ignore-platform --network-timeout 100000 --ignore-scripts",
+    "cmd": "yarn install  --ignore-engines --ignore-platform --network-timeout 100000 --ignore-scripts",
```
Comment (viceice) [inline lib/manager/npm/post-update/__snapshots__/yarn.spec.ts.snap:6]:
```suggestion
    "cmd": "yarn install --ignore-engines --ignore-platform --network-timeout 100000 --ignore-scripts",
```

File: lib/manager/npm/post-update/__snapshots__/yarn.spec.ts.snap [hunk 4]
```diff
@@ -25,7 +25,7 @@ Array [
 exports[`manager/npm/post-update/yarn generates lock files using yarn v1.22.0 1`] = `
 Array [
   Object {
-    "cmd": "yarn install --ignore-engines --ignore-platform --network-timeout 100000 --ignore-scripts",
+    "cmd": "yarn install  --ignore-engines --ignore-platform --network-timeout 100000 --ignore-scripts",
```
Comment (viceice) [inline lib/manager/npm/post-update/__snapshots__/yarn.spec.ts.snap:28]:
```suggestion
    "cmd": "yarn install --ignore-engines --ignore-platform --network-timeout 100000 --ignore-scripts",
```

File: lib/manager/npm/post-update/__snapshots__/yarn.spec.ts.snap [hunk 5]
```diff
@@ -59,7 +59,7 @@ Array [
     },
   },
   Object {
-    "cmd": "yarn install --ignore-engines --ignore-platform --network-timeout 100000 --ignore-scripts",
+    "cmd": "yarn install  --ignore-engines --ignore-platform --network-timeout 100000 --ignore-scripts",
```
Comment (viceice) [inline lib/manager/npm/post-update/__snapshots__/yarn.spec.ts.snap:62]:
```suggestion
    "cmd": "yarn install --ignore-engines --ignore-platform --network-timeout 100000 --ignore-scripts",
```

File: lib/manager/npm/post-update/__snapshots__/yarn.spec.ts.snap [hunk 6]
```diff
@@ -93,7 +93,7 @@ Array [
...
```
Comment (viceice) [inline lib/manager/npm/post-update/__snapshots__/yarn.spec.ts.snap:96]:
```suggestion
    "cmd": "yarn install --ignore-engines --ignore-platform --network-timeout 100000 --ignore-scripts",
```

File: lib/manager/npm/post-update/__snapshots__/yarn.spec.ts.snap [hunk 7]
```diff
@@ -138,7 +139,7 @@ Array [
 exports[`manager/npm/post-update/yarn performs lock file maintenance using yarn v1.22.0 1`] = `
 Array [
   Object {
-    "cmd": "yarn install --ignore-engines --ignore-platform --network-timeout 100000 --ignore-scripts",
+    "cmd": "yarn install  --ignore-engines --ignore-platform --network-timeout 100000 --ignore-scripts",
```
Comment (viceice) [inline lib/manager/npm/post-update/__snapshots__/yarn.spec.ts.snap:142]:
```suggestion
    "cmd": "yarn install --ignore-engines --ignore-platform --network-timeout 100000 --ignore-scripts",
```

File: lib/manager/npm/post-update/__snapshots__/yarn.spec.ts.snap [hunk 8]
```diff
@@ -172,7 +173,7 @@ Array [
     },
   },
   Object {
-    "cmd": "yarn install --ignore-engines --ignore-platform --network-timeout 100000 --ignore-scripts",
+    "cmd": "yarn install  --ignore-engines --ignore-platform --network-timeout 100000 --ignore-scripts",
```
Comment (viceice) [inline lib/manager/npm/post-update/__snapshots__/yarn.spec.ts.snap:176]:
```suggestion
    "cmd": "yarn install --ignore-engines --ignore-platform --network-timeout 100000 --ignore-scripts",
```

File: lib/manager/npm/post-update/__snapshots__/yarn.spec.ts.snap [hunk 9]
```diff
@@ -206,7 +207,7 @@ Array [
...
```
Comment (viceice) [inline lib/manager/npm/post-update/__snapshots__/yarn.spec.ts.snap:210]:
```suggestion
    "cmd": "yarn install --ignore-engines --ignore-platform --network-timeout 100000 --ignore-scripts",
```

File: lib/manager/npm/post-update/__snapshots__/yarn.spec.ts.snap [hunk 10]
```diff
@@ -251,7 +253,7 @@ Array [
 exports[`manager/npm/post-update/yarn performs lock file updates and full install using yarn v1.22.0 1`] = `
 Array [
   Object {
-    "cmd": "yarn install --ignore-engines --ignore-platform --network-timeout 100000 --ignore-scripts",
+    "cmd": "yarn install  --ignore-engines --ignore-platform --network-timeout 100000 --ignore-scripts",
```
Comment (viceice) [inline lib/manager/npm/post-update/__snapshots__/yarn.spec.ts.snap:256]:
```suggestion
    "cmd": "yarn install --ignore-engines --ignore-platform --network-timeout 100000 --ignore-scripts",
```

File: lib/manager/npm/post-update/__snapshots__/yarn.spec.ts.snap [hunk 11]
```diff
@@ -268,7 +270,7 @@ Array [
     },
   },
   Object {
-    "cmd": "yarn upgrade some-dep --ignore-engines --ignore-platform --network-timeout 100000 --ignore-scripts",
+    "cmd": "yarn upgrade some-dep  --ignore-engines --ignore-platform --network-timeout 100000 --ignore-scripts",
```
Comment (viceice) [inline lib/manager/npm/post-update/__snapshots__/yarn.spec.ts.snap:273]:
```suggestion
    "cmd": "yarn upgrade some-dep --ignore-engines --ignore-platform --network-timeout 100000 --ignore-scripts",
```

File: lib/manager/npm/post-update/__snapshots__/yarn.spec.ts.snap [hunk 12]
```diff
@@ -290,7 +292,7 @@ Array [
 exports[`manager/npm/post-update/yarn performs lock file updates using yarn v1.22.0 1`] = `
 Array [
   Object {
-    "cmd": "yarn install --ignore-engines --ignore-platform --network-timeout 100000 --ignore-scripts",
+    "cmd": "yarn install  --ignore-engines --ignore-platform --network-timeout 100000 --ignore-scripts",
```
Comment (viceice) [inline lib/manager/npm/post-update/__snapshots__/yarn.spec.ts.snap:295]:
```suggestion
    "cmd": "yarn install --ignore-engines --ignore-platform --network-timeout 100000 --ignore-scripts",
```

File: lib/manager/npm/post-update/__snapshots__/yarn.spec.ts.snap [hunk 13]
```diff
@@ -307,7 +309,7 @@ Array [
     },
   },
   Object {
-    "cmd": "yarn upgrade some-dep --ignore-engines --ignore-platform --network-timeout 100000 --ignore-scripts",
+    "cmd": "yarn upgrade some-dep  --ignore-engines --ignore-platform --network-timeout 100000 --ignore-scripts",
```
Comment (viceice) [inline lib/manager/npm/post-update/__snapshots__/yarn.spec.ts.snap:312]:
```suggestion
    "cmd": "yarn upgrade some-dep --ignore-engines --ignore-platform --network-timeout 100000 --ignore-scripts",
```

**Review by viceice — COMMENTED** [review]
will batch changes after this

**Review by viceice — APPROVED** [review]
LGTM, if ci is green

**Pull request merged**